tests/ui_tools/test_boxes.py

```diff
@@ -52,6 +52,7 @@ def test_init(self, write_box):
         assert write_box.model == self.view.model
         assert write_box.view == self.view
         assert write_box.msg_edit_state is None
+        assert not write_box.sent_start_typing_status
```

Comment:
This was my understanding of your suggestion to amend `test_init`, is this what you had in mind? If yes, should I add another commit asserting on other attributes initiated in the `write_box`?